services/github/github-issues.service.js

```diff
@@ -1,27 +1,15 @@
 import gql from 'graphql-tag'
 import Joi from 'joi'
-import { pathParams } from '../index.js'
+import { pathParams, queryParam } from '../index.js'
 import { metric } from '../text-formatters.js'
 import { nonNegativeInteger } from '../validators.js'
 import { GithubAuthV4Service } from './github-auth-service.js'
 import { documentation, transformErrors } from './github-helpers.js'
 
-const issueCountSchema = Joi.object({
```

Comment:
you seem to remove this schema while keeping the request for this later, why is it removed?